src/NuGet.Clients/NuGet.VisualStudio.Implementation/VsTemplateWizard.cs

```diff
@@ -463,7 +463,7 @@ void IWizard.RunStarted(object automationObject, Dictionary<string, string> repl
             // VsPackageInstaller and VsPackageUninstaller. Because, no powershell scripts get executed
             // as part of the operations performed below. Powershell scripts need to be executed on the
             // pipeline execution thread and they might try to access DTE. Doing that under
-            // ThreadHelper.JoinableTaskFactory.Run will consistently result in a hang
+            // ThreadHelper.JoinableTaskFactory.Run will consistently result in the UI stop responding
```

Comment:
replace "result in" with "make":
ThreadHelper.JoinableTaskFactory.Run will consistently make the UI stop responding.